proto/spacemesh/v1/mesh_types.proto

```diff
@@ -44,7 +44,7 @@ message LayerDurationResponse {
 message MaxTransactionsPerSecondRequest {}
 
 message MaxTransactionsPerSecondResponse {
-    SimpleInt maxtxpersecond = 1;
+    SimpleInt max_txs_per_second = 1;
```

Comment:
I'd rather not make superficial breaking changes like this, since it requires updating all downstream code...

Reply:
Better now than later... we just broke our own name convention here with naming this member. right?